lua/ibl/utils.lua

```diff
@@ -9,11 +9,23 @@ M.get_whitespace = function(line)
     return string.match(line, "^%s+") or ""
 end
 
+--- Use the faster validate version if available.
+---@param spec table<string, {[1]:any, [2]:function|string, [3]:string?, [4]:true?}>
+M.validate = function(spec)
+    if vim.fn.has("nvim-0.11") == 1 then
```

Comment:
I do confess that I'm not sure if this is enough since some early versions of 0.11 didn't yet have the new validation function, but I'm not too sure of how to more accurately check for this. Lmk @lukas-reineke if you have a better idea here.

Reply:
I also think this is what's causing the failing CI checks...